pennylane/qchem/openfermion_obs.py

```diff
@@ -1237,3 +862,35 @@ def _pyscf_integrals(
         two_mo = two_mo[qml.math.ix_(active, active, active, active)]
 
     return core_constant, one_mo, two_mo
+
+
+def _openfermion_hamiltonian(
```

Comment:
Is there a reason that the tests for molecular_hamiltonian using method as openfermion are not registering here?